dev-lisp/clisp/clisp-2.49.92-r2.ebuild

```diff
@@ -34,7 +34,7 @@ RDEPEND="
 	zlib? ( sys-libs/zlib )
 	X? ( x11-libs/libXpm )
 	hyperspec? ( dev-lisp/hyperspec )
-	berkdb? ( sys-libs/db:4.8 )
```

Comment:
What guarantees that it will link against this and not 4.8? Don't we need to pass some parameter to configure?

Reply:
(See also `db-use.eclass`, though I don't like the eclass very much.)

Reply:
I don't love it and in general, we should be careful about this -- but I suspect it has exactly the same problem now, or this might even make it better (as configure checks usually prefer the latest version available), so let's get this in to fix other problems.